vnext/ReactCommon/TEMP_UntilReactCommonUpdate/jsinspector-modern/NetworkIOAgent.cpp

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "NetworkIOAgent.h"
+#include <utility>
+#include "Base64.h"
+#include "Utf8.h"
+
+namespace facebook::react::jsinspector_modern {
+
+static constexpr long DEFAULT_BYTES_PER_READ =
+    1048576; // 1MB (Chrome v112 default)
+
+// https://github.com/chromium/chromium/blob/128.0.6593.1/content/browser/devtools/devtools_io_context.cc#L71-L73
+static constexpr std::array kTextMIMETypePrefixes{
+    "text/",
+    "application/x-javascript",
+    "application/json",
+    "application/xml",
+    "application/javascript" // Not in Chromium but emitted by Metro
+};
+
+// namespace { [Windows #13587]
```

Comment:
From what I was reading about anonymous-namespaces, this file should be able to scope to these variables but VS throws anonymous-namespace errors. Is there a better fix for this? 
![image](https://github.com/user-attachments/assets/20a0d2e7-22fa-4c4e-97ed-7722aa385f0f)

Reply:
not blocking the integration but will look more into this later :)